### PR TITLE
feature request #57

### PR DESCRIPTION
**Pull request by svecs132**

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const formatTime = (milliseconds: number, hideZeros = false): string => {
+  const days = Math.floor(milliseconds / (1000 * 60 * 60 * 24))
+  const hours = Math.floor((milliseconds % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60))
+  const minutes = Math.floor((milliseconds % (1000 * 60 * 60)) / (1000 * 60))
+  const seconds = Math.floor((milliseconds % (1000 * 60)) / 1000)
+
+  if (hideZeros) {
+    return `${days > 0 ? `${days}d ` : ''}${hours > 0 ? `${hours}h ` : ''}${minutes > 0 ? `${minutes}m ` : ''}${seconds}s`
+  } else {
```
Comment (Mezeman1) [inline src/utils/index.ts]: Else not necessary when returning

Comment (svecs132) [reply src/utils/index.ts]: yeah but to me it seems more clear that its one or the other when theryre on the same level and not "sequential" (even though it returns so not really but you know)

Comment (Mezeman1) [reply src/utils/index.ts]: I am a no-stacker, I do have some else statements here and there I plan on refactoring later.
But please, remove this else.

Comment (svecs132) [reply src/utils/index.ts]: okay, sorry

**Review by Mezeman1 — APPROVED** [review]

**Pull request merged**